tests/unit/deferred.py

```diff
@@ -140,23 +143,23 @@ def nic1():
                 host="ipmihost",
                 user="root",
                 password="tapeworm")),
-        'ipmi',
+        name,
         '00:11:22:33:44:55')
 
 
 @pytest.fixture()
+def nic1():
```

Comment:
Could we remove these as fixtures and inline these to the test itself?

I'm not sure that having them as fixtures helps unless we have additional tests beyond the first one.